src/Compute/custom/NewAzVmss_SimpleParameterSet.cs

```diff
@@ -0,0 +1,373 @@
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.PowerShell.Cmdlets.Compute.Models.Api20190301;
+using Microsoft.Azure.PowerShell.Cmdlets.Compute.Strategies;
+using Microsoft.Azure.PowerShell.Cmdlets.Compute.Support;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+
+namespace Microsoft.Azure.Commands.Compute.Automation
+{
+    [Cmdlet(VerbsCommon.New, ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "Vmss", DefaultParameterSetName = "DefaultParameter", SupportsShouldProcess = true)]
+    [OutputType(typeof(Microsoft.Azure.PowerShell.Cmdlets.Compute.Models.Api20190301.IVirtualMachineScaleSet))]
+    public partial class NewAzVmss_SimpleParameterSet : AzureRMAsyncCmdlet
+    {
+        public const string SimpleParameterSet = "SimpleParameterSet";
+
+        [Parameter(Mandatory = false, HelpMessage = "Run cmdlet in the background")]
+        public SwitchParameter AsJob { get; set; }
+
+        [Parameter(
+            ParameterSetName = SimpleParameterSet,
+            Mandatory = false)]
+        //[ResourceGroupCompleter]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Mandatory = true)]
+        [Alias("Name")]
+        public string VMScaleSetName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        [System.Management.Automation.ArgumentCompleter(typeof(Microsoft.Azure.PowerShell.Cmdlets.Compute.ImageName))]
+        Microsoft.Azure.PowerShell.Cmdlets.Compute.ImageName  Image { get; set; } = "Win2016Datacenter";
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = true)]
+        public PSCredential Credential { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public int InstanceCount { get; set; } = 2;
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string VirtualNetworkName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string SubnetName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string PublicIpAddressName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string DomainNameLabel { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string SecurityGroupName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public string LoadBalancerName { get; set; }
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        public int[] BackendPort { get; set; } = new[] { 80 };
+
+        [Parameter(ParameterSetName = SimpleParameterSet, Mandatory = false)]
+        //[LocationCompleter("Microsoft.Compute/virtualMachineScaleSets")]
```

Comment:
@markcowl same comment here -- we should remove this until we've implemented the `LocationCompleter` in the generated cmdlets